package.json

```
@@ -1,21 +1,31 @@
 {
-  "name": "begin-app",
+  "name": "tc39-ci",
   "version": "0.0.7",
-  "description": "Begin basic Hello World! app",
+  "description": "Publishes TC39 drafts and updates to S3",
   "scripts": {
     "lint": "eslint . --ignore-pattern node_modules --fix",
-    "start": "cross-env NODE_ENV=testing npx sandbox",
-    "test": "echo 'tests skipped' || cross-env NODE_ENV=testing PORT=6666 tape test/*-test.js ./test/**/*-test.js | faucet",
+    "start": "cross-env npx sandbox",
```

```suggestion
    "start": "npx sandbox",
```
we don't need cross-env if we're not setting env vars, but why aren't we setting NODE_ENV anymore?

---

In Arc 10 (2022) Arc removed all references to `NODE_ENV`; we found that too many other frameworks and tools were mutating it, so now it relies on its own `ARC_ENV` env var. Of course, everyone is still welcome to use `NODE_ENV` in their business logic, we just don't rely on that any longer.

---

what's wrong with other things mutating it? isn't "whatever it's set to" meant to be the right env value?